src/main/target/common_pre.h

```diff
@@ -92,6 +92,7 @@
 #define USE_TIMER_MGMT
 #define USE_PERSISTENT_OBJECTS
 #define USE_CUSTOM_DEFAULTS_ADDRESS
+#define USE_LATE_TASK_STATISTICS
```

Comment:
What is the reason for only having this in some of the targets?

Reply:
`USE_LATE_TASK_STATISTICS` adds extra stats into the task_t structure which we exclude for smaller target FLASH. STM32F405 build was failing with this defined at one point, so being rather code size sensitive, I made it optional. It can be enabled for specific F4 targets if needed for scheduler/task debug.